Content.Client/_Moffstation/ReadyManifest/UI/ReadyManifestEntry.xaml.cs

```diff
@@ -0,0 +1,30 @@
+using Content.Shared.Roles;
+using Robust.Client.AutoGenerated;
+using Robust.Client.GameObjects;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Prototypes;
+
+namespace Content.Client._Moffstation.ReadyManifest.UI;
+
+[GenerateTypedNameReferences]
+public sealed partial class ReadyManifestEntry : GridContainer
+{
+    public ReadyManifestEntry(
+        JobPrototype job,
+       (int High, int Medium, int Low) jobCount,
+        IPrototypeManager prototypeManager,
+        SpriteSystem spriteSystem)
```

Comment:
XAML controls ought to have no-param constructors.

Also ty for changing this from the upstream PR to actually have XAML instead of all being programmatic <3